cranelift-codegen/meta/src/cdsl/regs.rs

```diff
@@ -11,6 +11,7 @@ pub struct RegBank {
     pub names: Vec<&'static str>,
     pub prefix: &'static str,
     pub pressure_tracking: bool,
+    pub pinned_reg: Option<u8>,
```

Comment:
small quibble, shouldn't this be `Option<RegUnit>`? (which is u16, rather than u8)

it currently writes a literal out without a trailing `u8`, which is then inferred as `u16` when compiling the generated code, but sure would be an odd bug to track down :D

Reply:
Good catch, thanks.

Reply:
Actually, there's no concept of RegUnit in the meta code, only in the non-meta code. I'll make it a u16 though, just for future proofing...